apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/FeatureExtractorSamples.kt

```diff
@@ -45,7 +45,7 @@ class FeatureExtractorSamples {
                 .its({ age }) {
                     // introduces an expectation-group block
                     // all expectations are evaluated inside an expectation-group block; for more details:
-                    // https://github.com/robstoll/atrium#define-single-expectations-or-expectation-groups
+                    // https://github.com/robstoll/atrium#define-single-expectations-or-expectation-groups TODO: check link
```

Comment:
:+1: should be https://github.com/robstoll/atrium#define-single-expectations-or-an-expectation-group, I already did a search and replace in main